tests/test_dataloader.py

```diff
@@ -89,26 +93,38 @@ def test_drop_non_intersecting_removes_all_elements_with_empty_intersection(self
         dataloader = VerticalDataLoader(self.dataset, batch_size=100)
         intersection = []
 
-        dataloader.drop_non_intersecting(intersection, intersection)
+        dataloader.drop_non_intersecting(intersection)
 
         assert len(dataloader.dataloader1.dataset.data) == 0
         assert len(dataloader.dataloader1.dataset.ids) == 0
         assert len(dataloader.dataloader2.dataset.targets) == 0
         assert len(dataloader.dataloader2.dataset.ids) == 0
 
-    def test_datasets_have_same_ids_after_drop_non_intersecting(self):
-        dataloader = VerticalDataLoader(self.dataset, batch_size=128)
+    # def test_datasets_have_same_ids_after_drop_non_intersecting(self):
```

Comment:
Why was this test commented out? If we still want to keep it to uncomment it later, can we instead remove the `#` and rename it to `def xtest.....` the `x` at the start of the function name means `pytest` ignores this test